File: app/PFE-Back/src/main/java/com/example/PFE/Back/DTO/WeviooSuggestion.java
package com.example.PFE.Back.DTO;

import com.example.PFE.Back.Model.Dish;
import lombok.AllArgsConstructor;
import lombok.Data;

import java.util.List;
import java.util.Optional;
@Data
@AllArgsConstructor
public class WeviooSuggestion {
    private String SuggestionMessage;
    private WeatherDegreeAndIcon weatherDegreeAndIcon;
   private List<Dish> SuggestionDishs;
}

File: src/Models/WeviooSuggestion.ts
import { Dish } from "./Dish";
import { WeatherDegreeAndIcon } from "./WeatherDegreeAndIcon";

export class WeviooSuggestion {
    suggestionMessage: string;
    weatherDegreeAndIcon: WeatherDegreeAndIcon;
    suggestionDishs: Dish [];
  
    constructor(
      suggestionMessage: string,
      weatherDegreeAndIcon: WeatherDegreeAndIcon,
      suggestionDishes: Dish []
    ) {
      this.suggestionMessage = suggestionMessage;
      this.weatherDegreeAndIcon = weatherDegreeAndIcon;
      this.suggestionDishs = suggestionDishes;
    }
  }